language/Portuguese_Brazil/strings.po

```diff
@@ -20,871 +20,871 @@ msgstr ""
 
 #: src/emu/ui/auditmenu.cpp:111
 msgid "Audit in progress..."
-msgstr ""
+msgstr "Auditoria em progresso"
 
 #: src/emu/ui/barcode.cpp:72
 msgid "New Barcode:"
-msgstr ""
+msgstr "Novo c�digo de barras"
 
 #: src/emu/ui/barcode.cpp:76
 msgid "Enter Code"
-msgstr ""
+msgstr "Entrar com o c�digo"
 
 #: src/emu/ui/barcode.cpp:118
 msgid "Barcode length invalid!"
-msgstr ""
+msgstr "Tamanho do c�digo de barras inv�lido"
 
 #: src/emu/ui/cheatopt.cpp:78
 #, c-format
 msgid ""
 "Cheat Comment:\n"
 "%s"
-msgstr ""
+msgstr "Coment�rio do Truque:\n"
 
 #: src/emu/ui/cheatopt.cpp:91
 msgid "All cheats reloaded"
-msgstr ""
+msgstr "Todos os truques recarregados"
 
 #: src/emu/ui/cheatopt.cpp:122
 msgid "Autofire Settings"
-msgstr ""
+msgstr "Configura��es do Autodisparo"
 
 #: src/emu/ui/cheatopt.cpp:140
 msgid "Reset All"
-msgstr ""
+msgstr "Reiniciar Tudo"
 
 #: src/emu/ui/cheatopt.cpp:143
 msgid "Reload All"
-msgstr ""
+msgstr "Recarregar Tudo"
 
 #: src/emu/ui/cheatopt.cpp:259
 msgid "Autofire Status"
-msgstr ""
+msgstr "Situa��o do Autodisparo"
```

Comment:
The work "Status" is already highly well understood ammong technically literate Brazilians (and portuguese speakers in general). I would keep it as "Status do Autodisparo"